x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/server/esdocs.ts

```diff
@@ -50,10 +49,6 @@ export function esdocs(): ExpressionFunction<'esdocs', Filter, Arguments, any> {
         help: argHelp.fields,
         types: ['string'],
       },
-      metaFields: {
```

Comment:
Currently, it's not possible to retrieve metaFields since we switched to using ESSQL to back this function, so I removed this argument.